README.rst

```diff
@@ -2,46 +2,5 @@
  REANA-Cluster
 ===============
 
-.. image:: https://img.shields.io/pypi/pyversions/reana-cluster.svg
-   :target: https://pypi.org/pypi/reana-cluster
-
-.. image:: https://img.shields.io/travis/reanahub/reana-cluster.svg
-   :target: https://travis-ci.org/reanahub/reana-cluster
-
-.. image:: https://readthedocs.org/projects/reana-cluster/badge/?version=latest
-   :target: https://reana-cluster.readthedocs.io/en/latest/?badge=latest
-
-.. image:: https://img.shields.io/coveralls/reanahub/reana-cluster.svg
-   :target: https://coveralls.io/r/reanahub/reana-cluster
-
-.. image:: https://badges.gitter.im/Join%20Chat.svg
-   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
-
-.. image:: https://img.shields.io/github/license/reanahub/reana-cluster.svg
-   :target: https://github.com/reanahub/reana-cluster/blob/master/LICENSE
-
-About
------
-
-REANA-Cluster is a component of the `REANA <http://www.reana.io/>`_ reusable and
-reproducible research data analysis platform. REANA-Cluster takes care of
-deploying and managing cluster services on Kubernetes.
-
-Features
---------
-
-- deploy REANA cluster on Kubernetes
-- configure REANA cluster components
-- inspect status of REANA components
-
-Useful links
-------------
-
-- `documentation <https://reana-cluster.readthedocs.io/>`_
-- `releases <https://github.com/reanahub/reana-cluster/releases>`_
-- `known issues <https://github.com/reanahub/reana-cluster/issues>`_
-- `source code <https://github.com/reanahub/reana-cluster>`_
-
-For more information about the REANA reusable analysis platform, please see the
-`REANA project <http://www.reana.io>`_ home page and the general `REANA
-documentation <http://reana.readthedocs.io/>`_ pages.
+**29 July 2020** This repository is now deprecated in favor of
+Helm deployment covered in `reana <https://github.com/reanahub/reana>`_.
```

Comment:
What do you think of adding links to the new docs? If someone lands here is because:
1. Was using REANA-Cluster before, so is in need of deployment docs (link to deployment docs).
2. Ended up here for other reasons/links, loose focus, then it would be nice to have a link to the docs home.

Note on deployment docs: Currently one would have to choose http://docs.reana.io/development/deploying-locally/ and http://docs.reana.io/development/deploying-at-scale/ but maybe if we make deploying a folder, we can point to http://docs.reana.io/development/deploying/ which would contain an index of all types of deployments, what do you think?

Reply:
Sure, this was just a provisional version, wording can be improved.